src/dispatch/asyncio/fastapi.py

```diff
@@ -0,0 +1,108 @@
+"""Integration of Dispatch functions with FastAPI for handlers using asyncio.
+
+Example:
+
+    import fastapi
+    from dispatch.asyncio.fastapi import Dispatch
```

Comment:
FastAPI wraps [Starlette](https://www.starlette.io/) which is an [ASGI](https://asgi.readthedocs.io/en/latest/) framework. Should the asynchronous interface be the default for FastAPI (and should the user opt-in to the synchronous/blocking API)?

Reply:
I asked myself the same question... I know FastAPI is natively async, but their home page shows examples with non-async code (https://fastapi.tiangolo.com/), which makes me think it's probably a popular default.

The package name with `asyncio` is also self-explanatory, but using something like `dispatch.blocking.fastapi` would somewhat look like inventing a new name that may not be as clear to developers.

Tough call 🤷